### PR TITLE
Add a dynamic header feature

### PR DESCRIPTION
**Pull request by yongmin86k**
# Add a dynamic header feature

image

### CODE REVIEW

File: client/js/screens/Home/Home.js [hunk 1]
```diff
@@ -53,16 +58,46 @@ class Home extends Component {
...
+      this.animateHeader();
+    }
+  };
+
+  animateHeader = () => {
+    Animated.timing(this.state.headerAnimation, {
+      duration: 500,
+      toValue: this.state.isAnimated ? {x: 0, y: -88} : {x: 1, y: 0},
+    }).start();
+  };
```
Comment (siddharthparmar7) [inline client/js/screens/Home/Home.js]: what is 88? Please don't put magic numbers in your code. Use %s if possible to create variabels/consts.

File: client/js/screens/Home/Home.js [hunk 2]
```diff
@@ -192,7 +227,26 @@ class Home extends Component {
...
+            width: '100%',
+            height: paddingHeight,
+          }}>
+          <VibrancyView
+            blurType="dark"
+            blurAmount={2}
+            style={{
+              width: '100%',
+              height: '100%',
+            }}
```
Comment (siddharthparmar7) [inline client/js/screens/Home/Home.js]: Please move your styles to styles.js. 
`opacity: this.state.headerAnimation.x,
  top: this.state.headerAnimation.y,` are fine where they are.

File: client/js/screens/Home/HomeContainer.js [hunk 3]
```diff
@@ -1,27 +1,44 @@
...
     return {
       headerTransparent: true,
+      headerStyle: {
+        backgroundColor: 'transparent',
+      },
+      headerBackground: () => {
+        return (
+          <View>
+            <StatusBar barStyle="light-content" />
+          </View>
```
Comment (siddharthparmar7) [inline client/js/screens/Home/HomeContainer.js]: just return `StatusBar`. Please remove unnecessary wrapping of `View` component.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes.

File: client/js/screens/Home/Home.js (hunk 1 above)
Comment (yongmin86k) [reply client/js/screens/Home/Home.js]: It's the height of the header navigation and I replaced it with the variable~

File: client/js/screens/Home/Home.js (hunk 2 above)
Comment (yongmin86k) [reply client/js/screens/Home/Home.js]: Done

File: client/js/screens/Home/HomeContainer.js (hunk 3 above)
Comment (yongmin86k) [reply client/js/screens/Home/HomeContainer.js]: Done

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**